base/shell/explorer/traywnd.cpp

```diff
@@ -58,6 +58,57 @@ void appbar_notify_all(HMONITOR hMon, UINT uMsg, HWND hwndExclude, LPARAM lParam
 
 static const WCHAR szTrayWndClass[] = L"Shell_TrayWnd";
 
+struct WINDOWPOSBACKUPDATA
+{
+    HWND hwnd;
+    UINT x;
+    UINT y;
+    UINT cx;
+    UINT cy;
+};
```

Comment:
Why all these custom fields?
It would be easier (and more complete) to use `SetWindowPlacement` and `GetWindowPlacement`,
and just store the hwnd and the `WINDOWPLACEMENT`.
Then you can even skip the `IsIconic` call, and bail out when `WINDOWPLACEMENT.showCmd == SW_SHOWMINIMIZED`

Reply:
Good point ! I'll do it that way